src/util.rs

```diff
@@ -100,7 +109,24 @@ pub fn start_daemon<I, S>(program: &str, args: I) -> io::Result<()>
         I: IntoIterator<Item = S>,
         S: AsRef<OsStr>,
 {
-    Command::new(program).args(args).spawn().map(|_| ())
+    // As per issue 2128 the decision was made to make this call as
+    // similar to the Unix behaviour as possible.
+    //
```

Comment:
I think putting this part explicitly into a comment doesn't provide much benefit. If anyone is curious, `git blame` will suffice here.